packages/alfa-rules/src/sia-r15/rule.ts

```diff
@@ -40,9 +40,11 @@ export default Rule.Atomic.of<Page, Iterable<Element>, Question>({
             )
           );
 
-        const roots = iframes.groupBy((iframe) => iframe.root());
+        const roots = iframes.groupBy((iframe) =>
+          iframe.root({ nested: true })
+        );
```

Comment:
There's no need to group the iframes by root if the rule is to consider all iframes across the page as a single bag.

Reply:
🙈